include/libtorrent/ssl/context.hpp

```diff
@@ -165,9 +165,8 @@ class TORRENT_EXTRA_EXPORT context : public boost::asio::ssl::context
 
 	error_code set_verify_dn(std::string name, error_code& ec)
 	{
-		using namespace std::placeholders;
 		m_impl->expected_dn = std::move(name);
-		set_verify_callback(std::bind(&context::verify_dn, this, _1, _2), ec);
+		set_verify_callback(std::bind(&context::verify_dn, this, std::placeholders::_1, std::placeholders::_2), ec);
```

Comment:
This could probably use some local `use std::placeholders` !

Reply:
@alexandre-janniaux That's right, I used it at first but eventually reverted because it caused an ambiguity with `boost::placeholders` in some of the CI environment (for instance the macOS build on travis)...